src/resources/generated/billing_portal_configuration.rs

```diff
@@ -49,7 +47,6 @@ pub struct BillingPortalConfiguration {
     /// Set of [key-value pairs](https://stripe.com/docs/api/metadata) that you can attach to an object.
     ///
     /// This can be useful for storing additional information about the object in a structured format.
-    #[serde(default)]
     pub metadata: Metadata,
```

Comment:
Similar question to the `skip_serializing_if` example above. Do we know for sure that `stripe` always returns `{}` if `metadata` is empty instead of just leaving the field out? Otherwise this might cause new runtime errors